src/shared/utils/fileviewer.tsx

```diff
@@ -0,0 +1,99 @@
+import cs from 'classnames'
+
+// enum type from GraphQL
+export const LINE_TYPE = Object.freeze({
+  HIT: 'H',
+  MISS: 'M',
+  PARTIAL: 'P',
+})
+
+export const LINE_STATE = Object.freeze({
+  COVERED: 'COVERED',
+  UNCOVERED: 'UNCOVERED',
+  BLANK: 'BLANK',
+  PARTIAL: 'PARTIAL',
+})
```

Comment:
`l`: Would recommend swapping to `{ } as const` over `Object.freeze`, you can read more about it [here](https://www.totaltypescript.com/concepts/as-const).